### PR TITLE
Move the manifest() rule to the examples tree.

### PR DESCRIPTION
**Pull request by aiuto**
- It does nothing that useful,
- It relies on the older, get_licenses_info, API.

This is also a way to kick a test which is failing on CI but works locally on both linux and mac.

### CODE REVIEW

**Comment by aiuto** [comment]
And, I win.  Tests pass with this refactoring.

File: examples/manifest/manifest.bzl [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+load(
+    "@rules_license//rules:gather_licenses_info.bzl",
+    "gather_licenses_info",
+)
+load(
+    "@rules_license//rules/private:gathering_providers.bzl",
+    "TransitiveLicensesInfo",
+)
+
+def get_licenses_mapping(deps, warn = False):
```
Comment (Wyverald) [inline examples/manifest/manifest.bzl:25]: can we load this from compliance.bzl instead?

**Comment by aiuto** [comment]
The more I look at this, the more I realize the example makes no sense.
I have no idea what was going through my mind when I wrote it.
It still should move, but it needs a rewrite anyway.

**Comment by aiuto** [comment]
This is messier than I have time for right now.

**Pull request closed without merging**